### PR TITLE
DB-1669 Osxcrossbuild

### PR DESCRIPTION
**Pull request by joseph-cliqz**
Build Browser for OSX on Linux

### CODE REVIEW

File: Jenkinsfile.crossbuild [hunk 1]
```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env/ groovy
```
Comment (alver-cliqz) [inline Jenkinsfile.crossbuild]: Rename file from Jenkinsfile.crossbuild to something with platform specific. Also it's good practice to use same platform specific prefix with all related names. For example, "Jenkinsfile.macoslinux.crossbuild" or "Jenkinsfile.macos.crossbuild"

File: Jenkinsfile.crossbuild [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            sh """#!/bin/bash -l
+                set -e
+                set -x
+                ./magic_build_and_package.sh --clobber
+            """
+        }
+    }
+    stage('Stash artifacts'){
+        stash includes: 'obj/dist/*de.mac.dmg', name: 'dedmg'
+        stash includes: 'obj/dist/*US.mac.dmg', name: 'usdmg'
```
Comment (alver-cliqz) [inline Jenkinsfile.crossbuild]: please use full prefix for EN, it is "en-US"

File: Jenkinsfile.crossbuild [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+                withCredentials([[
+                    $class: 'UsernamePasswordMultiBinding',
+                    credentialsId: CQZ_AWS_CREDENTIAL_ID,
+                    passwordVariable: 'AWS_SECRET_ACCESS_KEY',
+                    usernameVariable: 'AWS_ACCESS_KEY_ID']]) {
+
+                    sh """#!/bin/bash -l 
+                        set -x
+                        ./magic_upload_files_crossbuild.sh
```
Comment (alver-cliqz) [inline Jenkinsfile.crossbuild]: Let's try to use same file as we have for normal build. You can use OSX_CROSS_BUILD env. variable to specify proper path (dist/bin or /opt/) in original "magic_upload_files.sh" script

File: Jenkinsfile.release [hunk 4]
```diff
@@ -169,8 +169,24 @@ def prepareBuildConfig() {
         } else {
           submitResultList.add([buildParams.job, job.id, 'obj/build_properties.json'])
         }
-    }
-
+    },
+    'crossbuild': {
```
Comment (alver-cliqz) [inline Jenkinsfile.release]: Please use more detailed naming, like "macos.crossbuild"

**Comment by alver-cliqz** [comment]
3. Please check build logs, something wrong with build symbols package, for example in cliqz-1.24.1.en-US.mac.crashreporter-symbols-full.zip archive. All bz2 files inside archive is empty (0 bytes)

File: Jenkinsfile.crossbuild (hunk 1 above)
Comment (joseph-cliqz) [reply Jenkinsfile.crossbuild]: Done

File: Jenkinsfile.crossbuild (hunk 2 above)
Comment (joseph-cliqz) [reply Jenkinsfile.crossbuild]: Done

**Pull request merged**